group/actions.go

```diff
@@ -0,0 +1,95 @@
+package group
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+
+	tcclient "github.com/taskcluster/taskcluster-client-go"
+	"github.com/taskcluster/taskcluster-client-go/queue"
+)
+
+type arguments map[string]interface{}
+
+// SubCommand represents the function interface of the task subcommand.
+type SubCommand func(credentials *tcclient.Credentials, args arguments) bool
+
+func runCancel(credentials *tcclient.Credentials, args arguments) bool {
```

Comment:
Maybe write some high level comments about what's going on with this method. Didn't know about go routines till I reviewed this haha